src/core/types/nav_group.ts

```diff
@@ -0,0 +1,28 @@
+/*
+ * Copyright OpenSearch Contributors
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+import { EuiIconType } from '@elastic/eui/src/components/icon/icon';
+
+export enum NavGroupType {
+  SYSTEM = 'system',
+}
+
+/** @public */
+export interface ChromeNavGroup {
+  id: string;
+  title: string;
+  description: string;
+  order?: number;
+  icon?: EuiIconType;
+
+  /**
+   * Groups with type of NavGroupType.SYSTEM will:
+   * 1. Always display before USE_CASE_GROUP.
+   * 2. Not be pickable within the workspace creation page.
+   *
+   * @default undefined indicates it is of type useCase
```

Comment:
Should this comment be adjusted a bit? Maybe the comment should be not in core, but at where the `type` is used.

Reply:
Sure, move the comment to the enum NavGroupType, and the place which will use this type is in another PR and still waiting for finalized design.